### ISSUE TITLE
Force Suggestion Bar

### ISSUE THREAD

**Issue by mrowwy** [AnySoftKeyboard/AnySoftKeyboard#3859]
It is becoming a serious problem for me but the suggestion bar is very often missing from the keyboard. 

I've looked through old issues and found this:

https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1331

Whether it is intended behavior or not, it is quite annoying to not be able to use suggestions while typing into pretty much any browser address bar or the google search field.

Is there any way to force this to be always present and working?

**Comment by alensiljak** [comment]
Perhaps that is what the apps ask for but SwiftKey doesn't seem to care much. It would be nice to have an option to always show the suggestions bar.

**Comment by tb06** [comment]
> Perhaps that is what the apps ask for but SwiftKey doesn't seem to care much. It would be nice to have an option to always show the suggestions bar.

With the availability to set for each app individually it could be much greater 😀

**Comment by tb06** [comment]
> Is Settings -> Language -> Show Sugestions not working for you?

Not for some apps and in my case I would appreciate to have suggestions when I input passwords in some apps

**Comment by mrowwy** [comment]
> Is Settings -> Language -> Show Sugestions not working for you?

No, unfortunately not. Browser address bars for one still are missing suggestions. So are most "search" fields.

**Comment by piknockyou** [comment]
Hi. Same here. Is there any solution?

**Comment by seismiqkris** [comment]
I switched to Futo keyboard

**Comment by PsySc0rpi0n** [comment]
Same here. According to IA, some apps, in my case, Instagram, sends a flag called TYPE_TEXT_FLAG_NO_SUGGESTIONS and possibly another called IME_FLAG_NO_PERSONALIZED_LEARNING (incognito mode that prevents dictionaries to "learn" from user input) an this seems to prevent ASK to show suggestion bar in some apps! No ideia if this is accurate but make sense. In my case, only Instagram message prevents ASK from showing this bar with word suggestions!

Samsung keyboard shows suggestions with no problem in Instagram messages! ASK shows suggestions anywhere else without problems, only problematic in Instagram!

Edited;
In Dev Tools we can see this:

<img width="635" height="1280" alt="Image" src="https://github.com/user-attachments/assets/2def7f66-3733-40dd-999e-ba24cb79383b" />

and then this:

<img width="635" height="1280" alt="Image" src="https://github.com/user-attachments/assets/08255b62-5fd6-42d9-9396-a0d9391b752a" />
